src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -970,15 +974,25 @@ public void onSessionAccept( JingleSession peerJingleSession,
         // Extract and store various session information in the Participant
         participant.setRTPDescription(answer);
         participant.addTransportFromJingle(answer);
-        participant.addSSRCsFromContent(answer);
-        participant.addSSRCGroupsFromContent(answer);
+
+        try
+        {
+            participant.addSSRCsAndGroupsFromContent(answer);
```

Comment:
This is an informational comment and not a suggestion to change something; in the bridge we refer to SSRCs + SSRC groups as _RTP Encodings_, i.e. we reconstruct the _RTP encodings_ of a participant based on the SSRCs and the SSRC groups that it has signaled. I think that in the future, RTP encodings will be assigned an RIDs and may optionally have an SSRC. This is the direction that both ORTC and WebRTC seem to be heading towards.